turftopic/base.py

```diff
@@ -62,22 +60,53 @@ def get_topics(
         return topics
 
     def _topics_table(
-        self, top_k: int = 10, show_scores: bool = False
+        self,
+        top_k: int = 10,
+        show_scores: bool = False,
+        show_negative: bool = False,
     ) -> list[list[str]]:
-        topics = self.get_topics(top_k)
-        columns = ["Topic ID", f"Top {top_k} Words"]
+        columns = ["Topic ID", "Positive"]
+        if show_negative:
+            columns.append("Negative")
         rows = []
-        for topic_id, terms in topics:
+        try:
+            classes = self.classes_
+        except AttributeError:
+            classes = list(range(self.components_.shape[0]))
+        vocab = self.get_vocab()
+        for topic_id, component in zip(classes, self.components_):
+            highest = np.argpartition(-component, top_k)[:top_k]
+            highest = highest[np.argsort(-component[highest])]
+            lowest = np.argpartition(component, top_k)[:top_k]
+            lowest = lowest[np.argsort(component[lowest])]
```

Comment:
one potential issue with this is that as far as i can see this won't constrain `highest` to be positive, and `lowest` to be negative. do we perhaps want to do that? or change the name of the columns to something like `top k words` and `bottom k words`?

Reply:
Mmmmm I see what you mean. The way I thought about it is more "positive definition" - "negative definition" kinda thing where it doesn't seem as awkward, but you're right this might throw some people off. I will look into a better way.